pkg/ingester/lifecycle_test.go

```diff
@@ -27,28 +28,36 @@ const (
 	aLongTime = 60 * time.Second
 )
 
+func init() {
+	al := promlog.AllowedLevel{}
+	al.Set("debug")
```

Comment:
Did you mean to leave this in?

Reply:
Its kinda useful, debugging tests is hard without.  But I'll remove it.